### PR TITLE
{bio}[intel/2019b] LAST v1045

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ba57d7adc49be04ada6b16fe4ef9a040 for a full test report.

File: easybuild/easyconfigs/l/LAST/LAST-1045-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                                     'last-split',
+                                     'last-split8',
+                                     'last-train',
+                                     'maf-convert',
+                                     'maf-cut',
+                                     'maf-join',
+                                     'maf-sort',
+                                     'maf-swap',
+                                     'parallel-fasta',
+                                     'parallel-fastq']],
```
Comment (boegel) [inline easybuild/easyconfigs/l/LAST/LAST-1045-intel-2019b.eb]: Spreading this out vertically is a bit much, no? As long as they're sorted alphabetically, this is fine imho:

```python
sanity_check_paths = {
    'files': ['bin/%s' % x for x in ['fastq-interleave', 'lastal', 'lastal8', 'lastdb', 'lastdb8', 'last-dotplot',
                                     'last-map-probs', 'last-merge-batches', 'last-pair-probs', 'last-postmask',
                                     'last-split', 'last-split8', 'last-train', 'maf-convert', 'maf-cut',
                                     'maf-join', 'maf-sort', 'maf-swap', 'parallel-fasta', 'parallel-fastq']],
    'dirs': [],
}

Comment (lexming) [reply easybuild/easyconfigs/l/LAST/LAST-1045-intel-2019b.eb]: no big deal, fixed :+1:

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/b3e147eaf45555d25783c56037975c54 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @lexming!

**Pull request merged**